client/my-sites/email/email-providers-stacked-comparison/provider-cards/google-workspace-card.tsx

```diff
@@ -174,13 +193,14 @@ const GoogleWorkspaceCard = ( {
 			return;
 		}
 
-		const domains: { name: string }[] = domain ? [ domain ] : [];
-
 		setAddingToCart( true );
 
 		const cartItems: MinimalRequestCartProduct[] = getItemsForCart(
-			domains,
-			gSuiteProduct.productSlug,
+			domains as {
+				name: string;
+				googleAppsSubscription?: { status?: string | undefined } | undefined;
+			}[],
```

Comment:
I think it would be much cleaner to define this using a new variable above, e.g. `const domainsForCart = ...`
The forcing function within the `getItemsForCart()` call makes this code do a _lot_ all at once.